client/cloudevents/protocol/fake/client.go

```diff
@@ -0,0 +1,48 @@
+package fake
+
+import (
+	"context"
+	"fmt"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+
+	"github.com/cloudevents/sdk-go/v2/event"
```

Comment:
`cloudevents.Event` is alias of `event.Event`: https://github.com/cloudevents/sdk-go/blob/main/v2/alias.go#L29
Can we just use `cloudevents.Event`?

Reply:
using `cloudevents.Event` instead